src/webgpu/api/validation/validation_test.ts

```diff
@@ -214,18 +214,19 @@ export class ValidationTest extends GPUTest {
     }
   }
 
+  /** Return a GPUBuffer with descriptor from mismatched device. */
+  getDeviceMismatchedBuffer(descriptor: GPUBufferDescriptor): GPUBuffer {
+    return this.trackForCleanup(this.mismatchedDevice.createBuffer(descriptor));
+  }
```

Comment:
Actually, even these helpers are not helping much. Doing just `this.mismatchedDevice.createBuffer` is not worse than `this.getDeviceMismatchedBuffer` (except you also want the trackForCleanup call).

Reply:
Yes, using `getDeviceMismatchedBuffer` and `createBufferWithState` is for the trackForCleanup call. In addition, we can easily modify the `mismatchedDevice` without updating case by case.

Reply:
Use `device.createBuffer` directly, and add `trackForCleanup`.